dbms/src/Client/QueryParts.h

```diff
@@ -0,0 +1,180 @@
+#ifndef CLICKHOUSE_QUERY_PARTS_H
+#define CLICKHOUSE_QUERY_PARTS_H
+
+typedef struct {
+    const char *name;			/* User printable name of the query part. */
+} QUERYPART;
+
+QUERYPART queryParts[] = {
+        // CREATE DATABASE, TABLE, VIEW
+        {(const char *)"CREATE"},
```

Comment:
Good. You can also compare with https://github.com/filimonov/chc/blob/8bb7b4a1a7917ff5cb75f84e7b76091463f23ecc/autocomplete.go#L12

Reply:
@filimonov Compared. Added missing query keywords